### PR TITLE
Add config for GN2 extended labelling training

### PR DESCRIPTION
**Pull request by neelamku**
classes added here: https://github.com/umami-hep/atlas-ftag-tools/pull/79

### CODE REVIEW

File: upp/configs/extended_labels.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+global_cuts: !include splits/simple-split.yaml
+
+# for k-folded splits, only required for final trainings of GN2
+# global_cuts: !include splits/fold0.yaml
+
+ttbar: &ttbar
+  name: ttbar
+  equal_jets: False
+  pattern:
+    - "user.svanstro.410470.e6337_s3681_r13144_p5922.tdd.EMPFlow_kfold.24_2_27.23-11-10_kfoldtrain_output.h5/*.h5"
```
Comment (samvanstroud) [inline upp/configs/extended_labels.yaml:11]: Just for my information - so far your studies only trained on ttbar?

**Review by samvanstroud — APPROVED** [review]
Thanks a lot @neelamku !

File: upp/configs/extended_labels.yaml (hunk 1 above)
Comment (neelamku) [reply upp/configs/extended_labels.yaml:11]: Yeah, uptil now all the trainings for both default and extended are only trained on ttbar

Comment (samvanstroud) [reply upp/configs/extended_labels.yaml:11]: Good to know, thanks!

**Pull request merged**